build.gradle

```diff
@@ -0,0 +1,29 @@
+allprojects {
+    apply plugin: "java"

```

Comment:
Where did you move the version and group?
You should define these parameters in the general build.gradle
```
group 'org.epam'
version '1.0'
```